tests/includes/juju.sh

```diff
@@ -302,6 +302,19 @@ pre_bootstrap() {
 		export BOOTSTRAP_ADDITIONAL_ARGS="${BOOTSTRAP_ADDITIONAL_ARGS:-} --config caas-image-repo=${OPERATOR_IMAGE_ACCOUNT}"
 	fi
 
+	if [[ ${BOOTSTRAP_PROVIDER:-} == "k8s" ]]; then
+		if [[ -n ${CONTROLLER_CHARM_PATH_CAAS:-} ]]; then
+			export BOOTSTRAP_ADDITIONAL_ARGS="${BOOTSTRAP_ADDITIONAL_ARGS:-} --controller-charm-path=${CONTROLLER_CHARM_PATH_CAAS}"
+		fi
+		if [[ -n ${CONTROLLER_CHARM_CHANNEL:-} ]]; then
+			export BOOTSTRAP_ADDITIONAL_ARGS="${BOOTSTRAP_ADDITIONAL_ARGS:-} --controller-charm-channel=${CONTROLLER_CHARM_CHANNEL}"
```

Comment:
Does this not work with IAAS?

Reply:
nope, because on IaaS, `--controller-charm-path` has to be a path on your local filesystem, whereas on CaaS, `--controller-charm-path` is a Charmhub URL